### PR TITLE
Added new section on comparators + edited formatting

### PR DESCRIPTION
**Pull request by nrosed**
- Added comparator text (last paragraph before recommendations)
- Rephrased the transparency section
- did some reformatting of the text to meet N.H.B. guidelines. 
    - @arielah I also added the full address for Center for Health AI, could you add in yours?
    - "Please list the full email address for each corresponding author in clear text"
    - TODO: need to reformat the references

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -6,32 +6,20 @@ keywords:
...
 lang: en-US
 authors:
-  - github: arielah
-    name: Ariel A. Hippen*
+  - name: Ariel A. Hippen*
     orcid: 0000-0001-9336-6543
-    twitter: ariel_hippen
     email: ariel.hippen@gmail.com
     affiliations:
       - Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania
```
Comment (arielah) [inline content/metadata.yaml]: My best guess is "Perelman School of Medicine, University of Pennsylvania, 3400 Civic Center Boulevard, Philadelphia, PA 19104"

File: content/01.main_content.md [hunk 2]
```diff
@@ -24,10 +26,19 @@ Proxy predictions of ethnicity via name origin are more difficult still; choosin
...
+  Choosing the correct reference distribution will be specific to the application, but the selection criteria of the background distribution should be annotated and justified.
+
 Recognizing the aforementioned shortcomings, we propose the following recommendations for the creation and deployment of automated auditing tools:
-  1. *Transparency.* Publicly provide all tools, code, and data used in the analysis.
-  This enables public scrutiny to those being audited and those whose data are used.
-  For confidential data, provide de-identified or aggregated data.
+
+  1. *Transparency.* Publicly provide all results, tools, code, and data used in the analysis.
+  This provides multiple benefits.
+  Firstly, this gives data access to those whose data was used in the analysis.
```
Comment (arielah) [inline content/01.main_content.md]: What do you think of "Publicly provide all results, tools, code, and data used in the analysis. This gives those whose data was used in the analysis, as well as the general public, the chance to scrutinize the methods used and be informed of any biases found by the audit. For confidential data, de-identified or aggregated data should be provided." ?

I think I asked you to make it wordier and then I changed my mind. :upside_down_face:

File: content/01.main_content.md [hunk 3]
```diff
@@ -24,10 +26,19 @@ Proxy predictions of ethnicity via name origin are more difficult still; choosin
   For instance, classifiers are usually unable to distinguish if names of Hispanic origin come from the Iberian Peninsula or from Latin America [@doi:10.1016/j.cels.2021.07.007].
   If a target and background population had identical probabilistic proportions of Hispanic names, but the target group primarily consisted of Iberian individuals, underrepresentation of Latin American scientists would go unnoticed in an exclusively name-based analysis.
 
+Beyond accurate classification of individuals' social identifiers, auditing systems also require a comparable background distribution.
+  For example, in Le et al. [@doi:10.1016/j.cels.2021.07.007], in order to identify disparities in ISCB honorees, they compared against scientists' publication records.
```
Comment (arielah) [inline content/01.main_content.md]: Is it weird to say "they" when there's overlap in the author lists of that paper and this one? More of a philosophical question than a suggestion we change it.

File: content/01.main_content.md [hunk 4]
```diff
@@ -24,10 +26,19 @@ Proxy predictions of ethnicity via name origin are more difficult still; choosin
...
+  While seemingly straightforward, there exist many choices on how to calculate publication rates. 
+  The authors had to decide which journals would be included in their publication record, whether the publications should be weighted by citation count, and if authorship position should be considered.
+  Choosing the correct reference distribution will be specific to the application, but the selection criteria of the background distribution should be annotated and justified.
```
Comment (arielah) [inline content/01.main_content.md]: I'd switch this to "The most appropriate reference distribution..."

**Review by arielah — COMMENTED** [review]
Looks good! I made a couple of suggestions, let me know what you think.

File: content/metadata.yaml (hunk 1 above)
Comment (nrosed) [reply content/metadata.yaml]: got it, added in https://github.com/greenelab/disparities_opinion/pull/16/commits/7133a790a31730fe67e5c6bc7d3e2b811e2d4b3f

File: content/01.main_content.md (hunk 2 above)
Comment (nrosed) [reply content/01.main_content.md]: yeah, i think that sounds better 👍🏼 
added in https://github.com/greenelab/disparities_opinion/pull/16/commits/22d18f281e53af29882742a2cc20e239cc0bb4c2

File: content/01.main_content.md (hunk 4 above)
Comment (nrosed) [reply content/01.main_content.md]: ah, yes, good point
https://github.com/greenelab/disparities_opinion/pull/16/commits/2021d8537658d4db25727b3608e952ecddaae4bc

File: content/01.main_content.md (hunk 3 above)
Comment (nrosed) [reply content/01.main_content.md]: hm yes, I reordered the sentence to remove the need for "they"
https://github.com/greenelab/disparities_opinion/pull/16/commits/a5b3c54b8466e3080e0e6d52c75cbec7f5528f53

File: content/01.main_content.md [hunk 5]
```diff
@@ -24,10 +26,17 @@ Proxy predictions of ethnicity via name origin are more difficult still; choosin
   For instance, classifiers are usually unable to distinguish if names of Hispanic origin come from the Iberian Peninsula or from Latin America [@doi:10.1016/j.cels.2021.07.007].
   If a target and background population had identical probabilistic proportions of Hispanic names, but the target group primarily consisted of Iberian individuals, underrepresentation of Latin American scientists would go unnoticed in an exclusively name-based analysis.
 
+Beyond accurate classification of individuals' social identifiers, auditing systems also require a comparable background distribution.
+  For example, in order to identify disparities in ISCB honorees, Le et al. [@doi:10.1016/j.cels.2021.07.007] used scientists' publication records as the expected honoree rate.
```
Comment (arielah) [inline content/01.main_content.md]: "expected honoree rate" is a weird phrasing. Maybe "used scientists' publication records to approximate the pool of potential honorees"?

**Review by arielah — APPROVED** [review]
Looks good, just one more comment from me.

File: content/01.main_content.md (hunk 5 above)
Comment (nrosed) [reply content/01.main_content.md]: hm... yes I can see it as a bit awkward.
What about:
"  For example, in order to identify disparities in ISCB honorees, Le et al. used scientists' publication records to approximate the expected diversity of potential honorees."

https://github.com/greenelab/disparities_opinion/pull/16/commits/018f34f2d05cc5c6e42e9c2a3cad0b53783973a2

**Pull request merged**